### PR TITLE
FTTD-4 A FrontEnd Page that displays a random activity

### PR DESCRIPTION
**Pull request by snaanortey**
New updates include:

- HTML file  `index.html` with frontend codes to display random activity description and references, where applicable
- JavaScript frontend file `frontend.js` for extracting response from body as JSON and using DOM to display in HTML file
- Backend JavaScript file `serveHTML.js` to parse GET HTTP request from app to HTML file 
- Import of CORS
- Update of comments and README file

### CODE REVIEW

File: frontend/frontend.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+and then, calss the populateDescription function  
+*/
+function processFetchedData(response) {
+  const serverResponse = response.json();
+  serverResponse.then(populateDescription);
+}
+
+// This function is to put description and references in html
+function populateDescription(data) {
+  document.getElementById("description").innerText = data.description;
```
Comment (ehimah) [inline frontend/frontend.js]: you should rename this function

```suggestion
function populateFunActivities(data) {
  document.getElementById("description").innerText = data.description;
```

File: index.js [hunk 2]
```diff
@@ -33,6 +39,8 @@ mongoose.connect(
 // Create an instance of an express app
 const server = express();
 
+server.use(cors());
+
```
Comment (ehimah) [inline index.js:44]: add a comment for this line

File: index.js [hunk 3]
```diff
@@ -44,6 +52,10 @@ server.use(randomActivity);
 // Register the random activity route to be used here
 server.use(suggestActivity);
 
+// Register the serve HTML route to be used here
+server.use(serveHTML);
+server.use('/frontend', express.static('frontend'))
+
```
Comment (ehimah) [inline index.js]: this too is lacking a comment

**Review by ehimah — COMMENTED** [review]
some comments to fix up

**Comment by ehimah** [comment]
yo! what's the update here?

File: frontend/frontend.js (hunk 1 above)
Comment (snaanortey) [reply frontend/frontend.js]: I have changed the function name

File: index.js (hunk 2 above)
Comment (snaanortey) [reply index.js:44]: Done

File: index.js (hunk 3 above)
Comment (snaanortey) [reply index.js]: Comment added

**Comment by snaanortey** [comment]
I have addressed the comments.

**Review by ehimah — APPROVED** [review]

**Pull request merged**